src/CodeGen_OpenGLCompute_Dev.cpp

```diff
@@ -157,10 +159,9 @@ void CodeGen_OpenGLCompute_Dev::CodeGen_OpenGLCompute_C::visit(const For *loop)
             const IntImm *int_limit = loop->extent.as<IntImm>();
             user_assert(int_limit != nullptr) << "For OpenGLCompute workgroup size must be a constant integer.\n";
             int new_workgroup_size = int_limit->value;
-            user_assert(workgroup_size[index] == 0 || workgroup_size[index] == new_workgroup_size) <<
-                "OpenGLCompute requires all gpu kernels have same workgroup size, "
-                "but two different ones were encountered " << workgroup_size[index] << " and " << new_workgroup_size <<
-                " in dimension " << index << ".\n";
+            user_assert(workgroup_size[index] == 0 || workgroup_size[index] == new_workgroup_size) << "OpenGLCompute requires all gpu kernels have same workgroup size, "
```

Comment:
Another weird one.

Reply:
Fixed. clang-format doesn't seem to remove line breaks, so I just added some extra breaks to tame this one.